libs/blocks/region-nav/region-nav.js

```diff
@@ -25,10 +30,33 @@ function decorateLink(link, config, path) {
   let { href } = link;
   if (href.endsWith('/')) href = href.slice(0, -1);
   link.href = `${href}${path}`;
+
+  link.addEventListener('mouseover', () => {
```

Comment:
While we don't usually add comments, should we briefly touch on what this does or link the MWPW/PR for context? This is one of the things someone will see in 1-2 years and think "????"

Reply:
I did have a similar thought, but I argued that Git Blame would point back to this PR containing all the details 🤷